pkg/kubectl/cmd/util/openapi/openapi.go

```diff
@@ -194,7 +194,7 @@ func (o *Resources) parseDefinition(name string, s spec.Schema) Kind {
 		Fields:           map[string]Type{},
 	}
 	if err != nil {
-		glog.Warning(err)
+		glog.V(2).Info(err)
```

Comment:
During testing I discovered for some types, GVK metadata is not defined, so emit this error on stdout which is bad. So didn't have an option but to bump it to higher verbosity.